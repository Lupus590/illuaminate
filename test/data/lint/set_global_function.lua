-- config: (lint (only var:set-global))
function f() end

local g
function g() end
