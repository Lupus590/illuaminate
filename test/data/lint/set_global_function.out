set_global_function.lua:[2:1-2:17]: Setting unknown global function "f". [var:set-global]
   │
 2 │ function f() end
   │ ^^^^^^^^^^^^^^^^
No errors and 1 warnings
